File: backend/App/Domain/Enums/ClientStatusEnum.php
<?php

declare(strict_types=1);

namespace App\Domain\Enums;

enum ClientStatusEnum: string
{
    case Default = 'default';
    case AtBarbershop = 'at_barbershop';
    case OnQueue = 'on_queue';
    case Waiting = 'waiting';
    case InService = 'in_service';
    case Attended = 'attended';
    case Paid = 'paid';
}

File: tests/domain/enums/client_status_enum.py
from enum import StrEnum


class ClientStatusEnum(StrEnum):
    DEFAULT = "default"
    AT_BARBERSHOP = "at_barbershop"
    ON_QUEUE = "on_queue"
    WAITING = "waiting"
    IN_SERVICE = "in_service"
    ATTENDED = "attended"
    PAID = "paid"
